packages/corvid-local-site/src/siteConverter.js

```diff
@@ -162,6 +170,7 @@ const localFileToDocument = file => {
 
 const editorDocumentToLocalDocumentFiles = siteDocument => {
   const localDocumentFiles = [
+    ...getPagesTsConfigs(siteDocument.pages),
```

Comment:
@nemni8 	why not use the same format of maping here and passing a single page each time ?

Reply:
i wanted it to be transparent to the siteConvertor weather it writes tsconfig files or not.. this function return array of tsconfig files to write, if corvid types is not available then return empty array..